test/src/nyc-bin.js

```diff
@@ -689,4 +689,39 @@ describe('the nyc cli', function () {
       done()
     })
   })
+
+  it('allows alternative high and low watermarks to be configured', function (done) {
+    var args = [
+      bin,
+      '--watermarks.lines=90',
+      '--watermarks.lines=100',
+      '--watermarks.statements=30',
+      '--watermarks.statements=40',
```

Comment:
@JaKXz that's my feeling; I think it's an advanced enough option that we're probably okay with sticking with the historical implementation.